lib/ecto/query/inspect.ex

```diff
@@ -155,7 +155,12 @@ defimpl Inspect, for: Ecto.Query do
         {value, _type} -> Macro.escape(value)
         _              -> {:..., [], nil}
       end
-    Macro.to_string {:^, [], [escaped]}
+    case Macro.to_string({:^, [], [escaped]}) do
```

Comment:
I think we should change the whole function to something like this:

```elixir
  defp expr_to_string({:^, _, [ix]}, _, _, %{params: params}) do
    case Enum.at(params || [], ix) do
      {value, _type} -> "^" <> inspect(value, char_lists: :as_lists)
      _              -> "^..."
    end
  end
```

Thoughts?